index.js

```diff
@@ -19,8 +19,15 @@ mongoose
     // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
-  .then(() => {
+  .then(async (self) => {
```

Comment:
here, the self is uneceesary :)
the if you are not using it no need to "declare it"